resources/lang/en/accounts.php

```diff
@@ -67,6 +67,7 @@
     'notifications' => [
         'title' => 'Notifications',
         'topic_auto_subscribe' => 'automatically enable notifications on new forum topics that you create',
+        'beatmapset_discussion_qualified_problem' => 'notification for new problem on qualified beatmaps',
```

Comment:
`Receive notifications for new problems on qualified beatmaps` or something

Reply:
I think what the options are for need explaining as well :pensive: